ci/docker_run.sh

```diff
@@ -17,11 +17,26 @@ fi
 
 echo "Running Docker CI build for '$IMAGE_NAME' "
 
-docker build  -t $IMAGE_NAME .
+# Remove old docker cid just in case
+rm -f docker_cid
+
+docker build -t $IMAGE_NAME .
 exit_code=$?; [[ $exit_code != 0 ]] && exit $exit_code
+
+cleanup() {
+  echo "Running cleanup..."
+  echo "Killing running containers..."
+  cat docker_cid | xargs docker kill 
```

Comment:
@andrewvc maybe we should rather use `docker rm -f -v` here in case your script is force aborted and `prune` may not be executed?